source/reference/mongosync-states.txt

```diff
@@ -0,0 +1,74 @@
+.. _c2c-states:
+
+====================
+``mongosync`` States
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+``mongosync`` enters different states depending on the requests it
+receives. This page describes the states that ``mongosync`` can be in.
+``mongosync`` is in a single state at a particular time. The possible
+API operations you can run depend on the state of ``mongosync``.
+
+View the Current State
+----------------------
+
+To view the current state of ``mongosync``, use the
+:ref:`/progress <c2c-api-progress>`. endpoint. The endpoint returns the
+state in the ``state`` field.
+
+.. _c2c-states-descriptions:
+
+State Descriptions
+------------------
+
+The following table describes each state and the operations you can run
+when ``mongosync`` is in a given state.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 30 20
+
+   * - State
+     - Description
+     - Possible API Operations
+
+   * - ``IDLE``
+     - ``mongosync`` is initialized and ready for a sync job to
+       begin.
+     - - ``POST`` :ref:`/start <c2c-api-start>`
+       - ``GET`` :ref:`/progress <c2c-api-progress>`
+
+   * - ``RUNNING``
+     - The sync process is currently running. In this state, data is
+       initially synced to the destination cluster, and subsequent
+       writes are applied.
```

Comment:
Consider:

cluster. Subsequent writes to the source cluster are applied to the destination cluster.